packages/components/src/public/QueryColumn.ts

```diff
@@ -46,6 +48,20 @@ export class QueryLookup extends Record({
             })
         );
     }
+
+    hasQueryFilters(): boolean {
+        return isAllSamplesSchema(this.schemaQuery);
```

Comment:
It doesn't seem like the right abstraction here that `QueryLookup` knows anything about certain types of schemas/lookups. Instead, if we need client-side declared filters at the lookup level we should apply them when constructing the lookups. Then, when we populate/hydrate these lookups we can do particular things and/or have a processor for handling these types of things.

I would recommend having a `filters: Filter.IFilter[]` property on `QueryLookup` that is hydrated elsewhere and then removing the redundant check of `hasQueryFilters()` since consumers of the query lookup can inspect the `filters` property directly.